CMakeLists.txt

```diff
@@ -85,7 +85,7 @@ set(ISMRMRD_VERSION_PATCH 0)
 set(ISMRMRD_VERSION_STRING ${ISMRMRD_VERSION_MAJOR}.${ISMRMRD_VERSION_MINOR}.${ISMRMRD_VERSION_PATCH})
 set(ISMRMRD_SOVERSION ${ISMRMRD_VERSION_MAJOR}.${ISMRMRD_VERSION_MINOR})
 
-set(ISMRMRD_XML_SCHEMA_SHA1 "22137b8bcbe7fe670019c61d3c645b6c9289641a")
+set(ISMRMRD_XML_SCHEMA_SHA1 "37a9f39f5d5e615ea67919064aef269938812519")
```

Comment:
You should bump the version when you update this.

Reply:
Fixed